snapcraft/commands/initialize.py

```diff
@@ -0,0 +1,88 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
Other than init -> initialize and the updated template, are there are changes in this file that I should look at? Or is the rest of it just from the move from snapcraft_legacy?

Reply:
Functionally, it's identical.  There are two minor changes under the hood:
1. `os.path` -> `pathlib.Path`
2. Using [`get_snap_project()`](https://github.com/snapcore/snapcraft/blob/ddd328f07f1aa64c63907b9ffbd5eafa11818dde/snapcraft/parts/lifecycle.py#L81) to locate an existing snapcraft.yaml